stories/components/SInputCheckbox.02_Indeterminate.story.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+import SInputCheckbox from 'sefirot/components/SInputCheckbox.vue'
+import { computed, reactive } from 'vue'
+
+const title = 'Components / SInputCheckbox / 02. Indeterminate'
+const docs = '/components/input-checkbox'
+
+const options = reactive([
+  { label: 'Book', value: 1, modelValue: false },
+  { label: 'Diamonds (x2)', value: 2, modelValue: false },
+  { label: 'Obsidian (x4)', value: 3, modelValue: false }
+])
+
+const control = computed({
+  get() {
+    return options.every((option) => option.modelValue)
+      ? true
+      : options.some((option) => option.modelValue)
+        ? 'indeterminate'
+        : false
```

Comment:
This would work the same. Might even be faster. Is a bit less readable though 😅

```suggestion
    let count = 0
    options.forEach((option) => option.modelValue && count++)
    return count === options.length ? true : count > 0 ? 'indeterminate' : false
```